pyanaconda/dbus/typing.py

```diff
@@ -56,6 +57,8 @@
 # Container types.
 # Use Tuple, Dict and List from typing.
 # Use Variant from pydbus and get_variant.
+# Use Structure instead of Dict[Str, Variant].
+Structure = Dict[Str, Variant]
```

Comment:
Wouldn't the `Variant` here be a cause of bad optimization and make the DBus API harder to read?